pkg/network/node_identity.go

```diff
@@ -89,8 +89,12 @@ func renderNetworkNodeIdentity(conf *operv1.NetworkSpec, bootstrapResult *bootst
 	// HyperShift specific
 	if hcpCfg := hypershift.NewHyperShiftConfig(); hcpCfg.Enabled {
 		webhookCAClient = client.ClientFor(names.ManagementClusterName)
-		webhookCALookup = types.NamespacedName{Name: "openshift-service-ca.crt", Namespace: hcpCfg.Namespace}
-		caKey = "service-ca.crt"
+
+		data.Data["CAConfigMap"] = hcpCfg.CAConfigMap
```

Comment:
Is `CAConfigMap` and `CAConfigMapKey` used anywhere in the managed network node identity yaml templates?

Reply:
I don't see it in the yaml templates when I do a search on that directory for either `NetworkNodeIdentityCABundle` or `openshift-service-ca.crt`